pom.xml

```diff
@@ -76,19 +76,30 @@
       <artifactId>commons-math3</artifactId>
       <version>${commons-math3.version}</version>
     </dependency>
-
-    <!-- optional deps for additional functionality -->
     <dependency>
       <groupId>org.apache.poi</groupId>
       <artifactId>poi</artifactId>
       <version>${poi.version}</version>
-      <optional>true</optional>
```

Comment:
These dependencies should be optional, if some code is explicitly requiring them, it should be fixed rather than marking them as required.  DataFrames should only need `poi` for excel import/export, `xchart` for the graphing functions, etc.